doc/articles/toc.yml

```diff
@@ -41,16 +41,14 @@
 
 - name: Samples & Tutorials
   items:
-    - name: Tutorial 1
```

Comment:
So it was agreed that we are not keeping the easy slider tutorial 1?

Reply:
Yes, see resolved comment